### PR TITLE
fixes for successful compilation of the latest version (debian 11, ubuntu 22.04)

### PR DESCRIPTION
**Pull request by DmDS**

### CODE REVIEW

**Review by husisusi — COMMENTED** [review]
Why is this deleted?
`cd ${cool_dir}/browser
cp npm-shrinkwrap.json.in npm-shrinkwrap.json
npm cache clean --force
npm install --update-binary --no-shrinkwrap
npm audit fix --force
npm shrinkwrap --dev
npm install --save`

File: bin/onlineBuild.sh [hunk 1]
```diff
@@ -46,13 +46,6 @@ echo ""
 
 sudo -Hu cool bash -c "./configure --enable-silent-rules --with-lokit-path=${cool_dir}/bundled/include --with-lo-path=${lo_dir}/instdir --with-max-connections=$cool_maxcon --with-max-documents=$cool_maxdoc --with-poco-includes=/usr/local/include --with-poco-libs=/usr/local/lib ${cool_configure_opts}" || exit 4
 # coolwsd+loleaflet take around 8.5/${cpu} minutes to compile on fast cpu
-cd ${cool_dir}/browser
```
Comment (husisusi) [inline bin/onlineBuild.sh:49]: Why is this deleted?

Comment (DmDS) [reply bin/onlineBuild.sh:49]: i previously added this code due to npm compilation errors. in the latest version of "cool " everything compiles fine without it.

**Comment by husisusi** [comment]
@DmDS Thank You.
